gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/DagActionReminderScheduler.java

```diff
@@ -77,28 +95,45 @@ public DagActionReminderScheduler(StdSchedulerFactory schedulerFactory)
    * @throws SchedulerException
    */
   public void scheduleReminder(DagActionStore.LeaseParams leaseParams, long reminderDurationMillis,
-      boolean isDeadlineReminder)
-      throws SchedulerException {
+      boolean isDeadlineReminder) throws SchedulerException {
+    DagActionStore.DagAction dagAction = leaseParams.getDagAction();
     JobDetail jobDetail = createReminderJobDetail(leaseParams, isDeadlineReminder);
-    Trigger trigger = createReminderJobTrigger(leaseParams.getDagAction(), reminderDurationMillis,
+    Trigger trigger = createReminderJobTrigger(dagAction, reminderDurationMillis,
         System::currentTimeMillis, isDeadlineReminder);
-    log.info("Reminder set for dagAction {} to fire after {} ms, isDeadlineTrigger: {}",
-        leaseParams.getDagAction(), reminderDurationMillis, isDeadlineReminder);
-    quartzScheduler.scheduleJob(jobDetail, trigger);
+    log.info("Going to set reminder for dagAction {} to fire after {} ms, isDeadlineTrigger: {}",
```

Comment:
let's prefix all of these logs with `DagActionReminderScheduler` so easy to find